app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/itemdecoration/StickyHeaderItemDecoration.kt

```diff
@@ -30,7 +30,7 @@ class StickyHeaderItemDecoration constructor(
         val resource = context!!.resources
 
         textPaint.apply {
-            typeface = Typeface.DEFAULT
+            typeface = ResourcesCompat.getFont(context, R.font.notosans_medium)
```

Comment:
Nice! It use Font Resource 😉 
We can use `ResourcesCompat.getFont(context, R.font.notosans_medium, callback)`.
But, Probably it is already cached.
I think this is ok